### PR TITLE
Pagination follow up PR

### PR DESCRIPTION
**Pull request by eiffelwong1**
Issue: #<Number>

## Summary (1-2 sentences)

follow up PR to #168 for pagination

## Details (reason and description of the changes)

- rm console log
- moved link button to link.tsx
- remove button

### CODE REVIEW

File: app/components/ui/forms.tsx [hunk 1]
```diff
@@ -84,41 +76,3 @@ export function TextArea({ label, ...props }: TextAreaProps) {
...
-
-type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement>;
-
-export function Button({ children, ...props }: ButtonProps) {
-  return (
-    <button {...props} className="border-2 bg-red-700 text-white rounded-lg p-2" type="submit">
-      {children}
-    </button>
-  );
-}
```
Comment (eiffelwong1) [inline app/components/ui/forms.tsx:96]: remove the button too. 
to reduce duplications.
make sure no other components using it.

**Review by thisistonydang — DISMISSED** [review]
Nice refactoring. Approved!

File: app/components/ui/links.tsx [hunk 2]
```diff
@@ -21,3 +21,31 @@ export function Link(props: LinkProps) {
...
+  ...props
+}: LinkButtonProps) {
+  return (
+    <Link
+      {...props}
+      to={to}
+      prefetch="intent"
+      className="border-2 bg-red-700 text-white rounded-lg p-3 w-full text-center"
+      preventScrollReset={preventScrollReset}
+      type="submit"
```
Comment (andrelandgraf) [inline app/components/ui/links.tsx:46]: NIT: can remove `type="submit"` as it's not a button!

```suggestion

```

**Review by andrelandgraf — DISMISSED** [review]
LGTM!

File: app/components/ui/forms.tsx [hunk 3]
```diff
@@ -1,13 +1,5 @@
...
-  type InputHTMLAttributes,
-  type ReactNode,
-  type TextareaHTMLAttributes,
-} from 'react';
-import { Link } from '@remix-run/react';
+import { useState, type InputHTMLAttributes, type ReactNode, type TextareaHTMLAttributes } from 'react';
 import clsx from 'clsx';
-import type { LinkProps } from '@remix-run/react';
 
 export const MAX_FILE_SIZE_MB = 8;
```
Comment (andrelandgraf) [inline app/components/ui/forms.tsx:4]:
```suggestion

```

**Review by CamilingJS — APPROVED** [review]

**Review by andrelandgraf — APPROVED** [review]

**Pull request merged**